README.md

```diff
@@ -0,0 +1,93 @@
+OpenMW
+======
+
+[![Build Status](https://travis-ci.org/OpenMW/openmw.svg?branch=coverity_scan)](https://travis-ci.org/OpenMW/openmw)
+
+[![Coverity Scan Build Status](https://scan.coverity.com/projects/3740/badge.svg)](https://scan.coverity.com/projects/3740)
+
+OpenMW is an attempt at recreating the engine for the popular role-playing game
+Morrowind by Bethesda Softworks. You need to own and install the original game for OpenMW to work.
+
+* Version: 0.34.0
+* License: GPL (see docs/license/GPL3.txt for more information)
+* Website: http://www.openmw.org
+* IRC: #openmw on irc.freenode.net
+
+Font Licenses:
+* DejaVuLGCSansMono.ttf: custom (see docs/license/DejaVu Font License.txt for more information)
+
+Getting Started
+---------------
+
+* [Official forums](https://forum.openmw.org/)
+* [Installation instructions](https://wiki.openmw.org/index.php?title=Installation_Instructions)
+* [Build from source](https://wiki.openmw.org/index.php?title=Development_Environment_Setup)
+* [Testing the game](https://wiki.openmw.org/index.php?title=Testing)
+* [How to contribute](https://wiki.openmw.org/index.php?title=Contribution_Wanted)
+* [Report a bug](http://bugs.openmw.org/projects/openmw) - read the [guidelines](https://wiki.openmw.org/index.php?title=Bug_Reporting_Guidelines) before submitting your first bug!
+* [Known issues] (http://bugs.openmw.org/projects/openmw/issues?utf8=%E2%9C%93&set_filter=1&f%5B%5D=status_id&op%5Bstatus_id%5D=%3D&v%5Bstatus_id%5D%5B%5D=7&f%5B%5D=tracker_id&op%5Btracker_id%5D=%3D&v%5Btracker_id%5D%5B%5D=1&f%5B%5D=&c%5B%5D=project&c%5B%5D=tracker&c%5B%5D=status&c%5B%5D=priority&c%5B%5D=subject&c%5B%5D=assigned_to&c%5B%5D=updated_on&group_by=tracker)
+
+The data path
+-------------
+
+The data path tells OpenMW where to find your Morrowind files. If you run the launcher, OpenMW should be able to pick up the location of these files on its own, if both Morrowind and OpenMW are installed properly (installing Morrowind under WINE is considered a proper install).
+
+Command line options
+--------------------
+
+<pre>
+Syntax: openmw <options>
```

Comment:
The syntax only shows up as "Syntax: openmw".  The "<options>" portion is not shown.